frappe/utils/__init__.py

```diff
@@ -156,6 +155,19 @@ def split_emails(txt):
 
 	return email_list
 
+def validate_url(txt, throw=False):
+	try:
+		url = urlparse(txt).netloc
+		if not url:
```

Comment:
Also, allow empty values on save.

Right now an error is thrown:
![image](https://user-images.githubusercontent.com/15175501/116671314-6cd96900-a9be-11eb-8c80-b80d7e3620aa.png)

![image](https://user-images.githubusercontent.com/15175501/116671420-87abdd80-a9be-11eb-9451-208b45b3eb6e.png)

Add exception for both these values